build/config.js

```diff
@@ -17,6 +17,12 @@ const banner = (name, version) => `
  * Released under the MIT license.
  */
 `
+const intro = `
+if (!this['process']) {
```

Comment:
@farfromrefug any reason you are not checking for `global.process`? In this case I believe it does the same thing, since we are in a global context - but still curious if there is something I'm not aware of! :)

Reply:
Well to be honest i am not sure how "process" is handled in Nativescript. What i know is that VueJs checks ```process``` and not ```global.process```. That's why i used this.

Reply:
@farfromrefug `process` is not defined in NativeScript by default, and the reason we're adding it manually is that some vue plugins check for `process.env` which then causes issues since it's trying to access property `env` of undefined. Checking if it exists on `global` and then adding to it ensures that both `global.process` and just `process` are defined.

Reply:
@rigor789 VueJs 2.6 also uses it. So you are saying that i should do?:
```
if (!global['process']) {
  global.process = process = {env:{}}
}
```

Reply:
@farfromrefug yeah that looks good to me, just change `this['process']` to `global['process']`

Reply:
fixed, testing ```global.process```